src/Customer.php

```diff
@@ -115,6 +115,19 @@ public function lineItems()
         return $line;
     }
 
+    /**
+     * Gets a payment source object for this customer.
+     *
+     * @return PaymentSource
+     */
+    public function paymentSources()
+    {
+        $source = new PaymentSource($this->_client);
+        $source->setEndpointBase($this->getEndpoint());
```

Comment:
As a matter of style you could write this as `return $source->setEndpointBase(...)`